global-node-modules/deb/debian/changelog

```diff
@@ -1,3 +1,9 @@
+global-node-modules (0.9.0-1) unstable; urgency=medium
```

Comment:
Shouldn't this be SHA-based since we're referencing a SHA in build.sh?

Reply:
You're right that it's SHA based in build.sh.  `git describe` will omit the SHA if the commit lands exactly on the tag, so the `git describe` for this component version is `v0.9.0`.  Using our convention, we remove the "v" and add the "-1" to make "0.9.0-1".

It is possible to tell `git describe` to always include the SHA if we use the `--long` option.  I felt that a clean version string would be simpler to read.

Reply:
> git describe for this component version is v0.9.0.
Not true because there's no 0.9.0 tag in the devjs-production-tools repo.  I would agree that it would normally be something like `<tag>-1` or `<closest-tag>-g<SHA>-1` if the SHA doesn't match a tag, but 0.9.0 has no meaning in the underlying repo.

Reply:
I'm using edge-node-modules to do git describe.  This one is a bit different since it's two git repositories.

Reply:
ok I see now.  I missed the 2nd (first?) repo.